lexicons/tools/ozone/set/add.json

```diff
@@ -0,0 +1,31 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.add",
```

Comment:
To date there's been a bit of a convention going that lexicon methods are "verb-subject", which serves to distinguish lexicon methods from e.g. records or defs (which are always lowercase).  There aren't any lexicons methods that are all lowercase that I know of.  Should we keep to that convention?  I think it would become something like `sets.addElement` or `sets.addItem`.

@bnewbold I think you may have proposed these lexicons, so tagging you in!

Reply:
Another small one— to date there aren't any pluralized namespaces either, we could use `tools.ozone.set` if we wanted to continue with that.

Reply:
yeah I was going off the proposed lexicons by bryan but yeah, agreed that it would be more consistent to name it something like `addItems` or `addValues`

Reply:
ah, you are right about `verbSubject`. I agree we should reserve "single word lower-case" for records ("nouns") not endpoints ("verbs"). so probably `addValues` here. that makes the `remove` situation clearer as well.

for `tools.ozone.set`, "set" as a word is a bit weird (both a noun and verb in English) and "sets" clarifies that a bit. I would probably lean devin's way and prefer singular though.